omero/developers/index.html

```diff
@@ -68,8 +68,8 @@ <h4>OMERO.cli</h4>
             <div class="medium-1 columns"><img class="omero-feature thumbnail" alt="OMERO feature" src="{{ site.baseurl }}/img/icons/omero-features-icons_scripts.svg"></div>
             <div class="omero-key-feature-container medium-3 columns end">
                 <h4>Scripting Service</h4>
-                <p>Write Python scripts to add functionality to the OMERO.server. <a href="https://www.openmicroscopy.org/site/community/scripts" target="_blank">Share your scripts</a> with the community.</p>
-                <a class="button hollow tiny" href="https://docs.openmicroscopy.org/omero/{{ site.omero.version }}/developers/index.html#omero-scripts-plugins-for-omero" target="_blank">View Developer Guide</a>
+                <p>Write Python scripts to add functionality to the OMERO.server. <a href="https://github.com/ome/omero-user-scripts/network/members" target="_blank">Share your scripts</a> with the community.</p>
```

Comment:
This link is not correct
it should point to https://github.com/openmicroscopy/omero-example-scripts
where we have the examples scripts from external

Reply:
The link was as I intended but if you prefer that one, I'll fix it today

Reply:
The repo has no script  and no activity for 4 years so I think it is better to point to the suggested link

Reply:
Good point, sorted now